BackEnd/src/main/java/com/unosquare/admin_core/back_end/repository/EventRepository.java

```diff
@@ -15,9 +18,26 @@
 
     Event findByEmployeeAndStartDateAndEndDateAndEventType(Employee employee, LocalDate startDate, LocalDate endDate, EventType eventType);
 
-    List<Event> findByStartDateBetweenAndEventType(LocalDate rangeStart, LocalDate rangeEnd,EventType eventType);
+    List<Event> findByStartDateBetweenAndEventType(LocalDate rangeStart, LocalDate rangeEnd, EventType eventType);
 
     List<Event> findByEventStatusAndEventType(EventStatus eventStatus, EventType eventType);
 
     List<Event> findByEventType(EventType eventType);
+
+    @Query(value = "SELECT totalHolidays " +
+            "FROM Employee " +
+            "WHERE employeeId = :employeeId")
+    Long getTotalEventsRemainingByEmployee(@Param("employeeId") int employeeId);
```

Comment:
Does this belong on the event repository? We're querying from employee table

Reply:
Changed to event repository there. Didn't notice I had it in the wrong repo

Reply:
One thing to keep in mind, this number does not decrement. It is a set value that represents the available holidays for the year.

the holidays remaining = totalholidays - (sum of holiday events).

Reply:
Ok but is there a value that decrements each time an event is created?